forge/forge.js

```diff
@@ -314,7 +314,8 @@ module.exports = async (options = {}) => {
             strictTransportSecurity,
             frameguard: {
                 action: 'deny'
-            }
+            },
+            referrerPolicy: 'origin-when-cross-origin'
```

Comment:
The docs for helmet suggest this needs to be an object, not just a plain string. Have you verified this sets the header as you expect?

Also need to verify this will still send useful information to posthog/hubspot as they depend on the referer information.

Reply:
And yes it was setting the header, but I've updated to match the doc.

Reply:
Tested posthog, no email reset tokens in the referrer header, but the full page paths are still available in the posthog logs